pkg/azurefile/controllerserver.go

```diff
@@ -884,8 +904,8 @@ func (d *Driver) DeleteSnapshot(ctx context.Context, req *csi.DeleteSnapshotRequ
 		return nil, status.Error(codes.InvalidArgument, "Snapshot ID must be provided")
 	}
 
-	shareURL, err := d.getShareURL(ctx, req.SnapshotId, req.GetSecrets())
-	if err != nil {
+	_, _, _, fileShare, _, _, err := d.GetAccountInfo(ctx, req.SnapshotId, req.GetSecrets(), map[string]string{}) //nolint:dogsled
```

Comment:
I think you want to use `GetFileShareInfo` here instead of `GetAccountInfo`, this is only for parsing volumeID

Reply:
and L922 already did this

Reply:
This step is for CSI Driver Sanity Tester, invalid snapshot id used should succeed. However, `GetFileShareInfo` err should be return for status.Error. I didn't find a suitable function to replace `d.getShareURL`